scripts/pcm-dpc/covid-19/generate_statvar.py

```diff
@@ -0,0 +1,50 @@
+"""generate the mcf for StatisticalVariables"""
+
```

Comment:
If after reusing many of the existing StatVars, we may just decide to handwrite the StatVars (and remove this script)

Reply:
removed